Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/lucene/impl/LuceneUtility.java

```diff
@@ -622,15 +653,20 @@ private void buildQuery(BooleanClause.Occur occur, PerFieldAnalyzerWrapper perFi
 
     private void buildTokenizedOrWildCardQuery(LuceneField luceneFieldAnnotation, String searchString,
             BooleanQuery.Builder tokenizedQueryBuilder) {
-        for (String searchValue : searchString.split(" ")) {
+        String hypSearchStr = new String(searchString);
+        hypSearchStr = searchString.replace('-', ' ');
             if (luceneFieldAnnotation.isWildCardSearchEnabled()) {
+                for (String searchValue : hypSearchStr.split(" ")) {
                 createWildCardSearchQuery(luceneFieldAnnotation, searchValue, tokenizedQueryBuilder, Occur.MUST);
+                }
             } else {
+                for (String searchValue : hypSearchStr.split(" ")) {
                 tokenizedQueryBuilder.add(new PhraseQuery(luceneFieldAnnotation.lucenefieldName(), searchValue),
                         Occur.MUST);
+                }
```

Comment:
why is this still here? this should not be needed with the custom anlayzer.

Reply:
we talked about not instantiating query objects, but use the builder to be able to add the analyzers per field. Why did you do that? Was there a problem?

Reply:
Yes, we discussed about instantiating the objects using builder so that analyzer can be applied to the fields and I am working on it. This pull request was raised before our discussion, so it wasn't implemented.